### PR TITLE
WT-7931 Test_hs18 test_modifies failure.

### PR DESCRIPTION
**Pull request by Siddhartha8899**

### CODE REVIEW

**Comment by Siddhartha8899** [comment]
After going through the test, I believe the reason an assertion error occured on `self.check_value(cursor_ts_reader, values[1])` for the **test_hs18** **test_modifies()** was because WiredTiger was not evicting the modify at key 1.
Therefore,  I used an evict cursor on key 1 before the assertion and the test did not fail.

File: test/suite/test_hs18.py [hunk 1]
```diff
@@ -485,11 +481,13 @@ def test_modifies(self):
         # Start a long running transaction which could see modify 1.
         self.start_txn(sessions, cursors, values, 2)
 
-        # Insert a bunch of contents to fill the cache
-        for i in range(2000, 10000):
-            self.session.begin_transaction()
-            cursor[self.create_key(i)] = value_junk
-            self.session.commit_transaction()
+        # Evict the update using debug cursor
```
Comment (luke-pearson) [inline test/suite/test_hs18.py:484]: "Evict the update using a debug cursor"

**Review by luke-pearson — APPROVED** [review]
LGTM (with a grammar comment), make sure you run it in a loop for a while to make sure that the column store failure doesn't reproduce.

**Review by kommiharibabu — APPROVED** [review]
LGTM

**Pull request merged**